tools/src/commands/gen_certs/key_pair.rs

```diff
@@ -0,0 +1,29 @@
+use super::certificate_builder::CertificateBuilder;
+use anyhow::Result;
+use rcgen::Certificate;
+
+pub struct KeyPair(pub Vec<u8>, pub Vec<u8>);
+
+impl KeyPair {
+    pub fn client(ca: &Certificate) -> Result<Self> {
+        let cert_builder = CertificateBuilder::client();
+        Self::build(cert_builder, ca)
+    }
+
+    pub fn server(ca: &Certificate) -> Result<Self> {
+        let cert_builder = CertificateBuilder::server();
+        Self::build(cert_builder, ca)
+    }
+
+    fn build(builder: CertificateBuilder, ca: &Certificate) -> Result<Self> {
+        let cert = builder
+            .common_name("selium.com")
```

Comment:
This perhaps should be called out in a constant, and also use `selium.io` instead. We've got both domains and I've been thinking `.io` could be used for SaaS components.